src/test/java/org/sagebionetworks/bridge/services/Exporter3ServiceTest.java

```diff
@@ -1550,6 +1569,101 @@ public void completeUpload_NoAccount() throws Exception {
         verifyZeroInteractions(mockHealthDataEx3Service, mockSqsClient);
     }
 
+    @Test
+    public void exportTimelineForStudy_1stTime() throws Exception {
+        // Mock Study.
+        study.setScheduleGuid(TestConstants.SCHEDULE_GUID);
+
+        // Study has no exporter3config.
+        study.setExporter3Configuration(null);
+        study.setExporter3Enabled(false);
+
+        mockSynapseResourceCreation();
+
+        // Mock Timeline
+        Timeline.Builder builder = new Timeline.Builder();
+        Timeline timeline = builder.withSchedule(mockSchedule2Service.getSchedule(TEST_APP_ID, TestConstants.SCHEDULE_GUID)).build();
```

Comment:
It's not necessary to call mockSchedule2Service.getschedule(). Instead, we can just create a Schedule2 object directly.